doc/whats_new/v0.20.rst

```diff
@@ -652,6 +652,10 @@ Linear, kernelized and related models
   :class:`linear_model.LogisticRegression` when ``verbose`` is set to 0.
   :issue:`10881` by :user:`Alexandre Sevin <AlexandreSev>`.
 
+- In both :class:`ensemble.RandomForestClassifier` and `ensemble.ExtraTreesClassifier`,
```

Comment:
missing ``:class:`` for the extra trees. Also, the message should read something like " ``n_estimators`` will be changed to 100 in version 0.22. A FutureWarning is raised when the default value is used." or similar. ``n_estimators=10``doesn't raise a warning (as is tested).